feat(Credentials): Add doc & command to update credentials (#1321)
cozy/cozy-banks: docs/credentials.md

# Credentials

> All credentials are encrypt on archive file in `scripts/encrypted.tar.gz.enc`.

Reviewer: ```suggestion
All credentials are encrypted in an archive file in `scripts/encrypted.tar.gz.enc`.
```

## Create encrypted file

```sh
tar zcvf scripts/decrypted.tar.gz scripts/decrypted/
travis encrypt-file scripts/decrypted.tar.gz scripts/encrypted.tar.gz.enc -p
```

You should save `key` and `iv` variables on pass.

```sh

> pass insert Gangsters/cozy-banks/ENCRYPTED_KEY

Reviewer: ```suggestion
pass show Gangsters/cozy-banks/ENCRYPTED_KEY
```

> pass insert Gangsters/cozy-banks/ENCRYPTED_IV

Reviewer: ```suggestion
pass Gangsters/cozy-banks/ENCRYPTED_IV
```

```

## Decrypt file

```sh
export ENCRYPTED_KEY=`pass Gangsters/cozy-banks/ENCRYPTED_KEY`
export ENCRYPTED_IV=`pass Gangsters/cozy-banks/ENCRYPTED_IV`
yarn secrets:decrypt
```

## Update encrypt file

> If you want update credentials you should update folder `./scripts/decrypted`

Reviewer: ```suggestion
If you want to update the credentials, you should update folder `./scripts/decrypted`
```

> and launch this commands:

Reviewer: ```suggestion
and launch these commands:
```
